src/components/form-elements/_select.scss

```diff
@@ -43,35 +46,136 @@ $includeHtml: false !default;
       &:hover,
       &:focus,
       &:active {
-        border-color: $selectActiveBorderColor;
+        border-color: $graySecondaryLightest;
       }
     }
 
     &__icon {
       position: absolute;
-      top: $selectHeight / 2 - $selectIconSize / 2.5;
-      right: gutter(1 / 2);
-      height: $selectIconSize;
-      width: $selectIconSize;
+      top: spacing(xs) - 2px;
+      right: spacing(xs);
       pointer-events: none;
       z-index: 1;
       display: flex;
-      border-left: 4px solid transparent;
-      border-right: 4px solid transparent;
-      border-top: 6px solid $arrowColor;
+    }
+
+    &--white {
+      background: $white;
+      border: 2px solid $white;
+
+      .sg-select__element {
+        background: $white;
+        border: 2px solid $white;
+
+      }
+    }
 
+    &.sg-select--white {
+      &.sg-select--valid {
+        &:hover {
+          background-color: mix($white, $graySecondaryUltraLight, 12%);
+          border: 2px solid $selectActiveValidBorderColor;
+        }
+      }
+
+      &.sg-select--invalid {
+        &:hover {
+          background-color: mix($white, $graySecondaryUltraLight, 12%);
+          border: 2px solid $selectActiveInvalidBorderColor;
+        }
+      }
+      
+      &:hover {
+        background-color: mix($white, $graySecondaryUltraLight, 12%);
+        border: 2px solid mix($white, $graySecondaryUltraLight, 12%);
+  
+        .sg-select__element {
+          background-color: mix($white, $graySecondaryUltraLight, 12%);
+          border: 2px solid mix($white, $graySecondaryUltraLight, 12%);
+        }
+      }
+    }
+
+    &:hover {
+      background-color: mix($selectBackground, $graySecondaryLight, 80%);
+      border: 2px solid mix($selectBackground, $graySecondaryLight, 80%);
+
+      .sg-select__element {
+        background-color: mix($selectBackground, $graySecondaryLight, 80%);
+        border: 2px solid mix($selectBackground, $graySecondaryLight, 80%);
+      }
     }
 
     &--full-width {
       width: 100%;
     }
 
+    &--large {
+      height: $selectLargeHeight;
+      font-size: $selectLargeFontSize;
+      border-radius: $selectLargeBorderRadius;
+
+      .sg-select__icon {
+        right: spacing(s);
+        top: spacing(s)
+      }
+
+      .sg-select__element {
+        position: relative;
+        top: -2px;
+        border-radius: $selectLargeBorderRadius;
+        font-size: $selectLargeFontSize;
+        height: $selectLargeHeight;
+        padding: 0 50px 0 spacing(m);
+        
+        &::-ms-expand {
+          display: none;
+        }
+      }
+
+      &::placeholder {
+        font-size: 18px;
```

Comment:
why don't we reuse variables? like this `$inputLargeFontSize` - its 18px

Reply:
I am moving it into configuration. I will let you know as soon as it's refactored